### PR TITLE
Desktop notifications for Linux

### PR DESCRIPTION
**Pull request by r-darwish**

### CODE REVIEW

**Comment by r-darwish** [comment]
@idanski - can you please test this change in Linux? I don't have GUI in my VM.

**Comment by idanski** [comment]
@r-darwish 
Sure. I'll try to do it later today/tomorrow.

Thanks!

File: src/terminal.rs [hunk 1]
```diff
@@ -59,23 +69,41 @@ impl Terminal {
...
+                if let Some(timeout) = timeout {
+                    notification.timeout(Timeout::Milliseconds(timeout.as_millis() as u32));
+                }
+                notification.show().ok();
+            } else if #[cfg(target_os = "linux")] {
+                if let Some(ns) = self.notify_send.as_ref() {
+                    let mut command = Command::new(ns);
+                    if let Some(timeout) = timeout {
+                        command.arg("-t");
+                        command.arg(format!("{}", timeout.as_millis()));
```
Comment (idanski) [inline src/terminal.rs]: I think you forgot to add the message :)

**Comment by idanski** [comment]
Screenshot_20200303_132030

@r-darwish it works but I suggest adding param `-a Topgrade` to have a good title instead of the default notify-send.

**Pull request merged**